.github/workflows/run-tests.yml

```diff
@@ -337,24 +337,21 @@ jobs:
           echo CTEST_BUILD_NAME: ${CTEST_BUILD_NAME}
 
       - name: Build Docker Container with Testing Environment
-        id: docker-build
-        uses: whoan/docker-build-with-cache-action@v5
-        with:
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-          registry: docker.pkg.github.com
-          image_name: qgis3-qt${{ matrix.qt-version }}-build-deps-bin-only
```

Comment:
why not pushing and using the matrix to determine the image name?

Reply:
#44640